app/components/proposal_details/groupable.rb

```diff
@@ -27,13 +27,13 @@ def title
     end
 
     def name
-      case portal_name
+      case section_name
       when "_root"
```

Comment:
Is this identifier still going to be the same under the new structure?

Reply:
lol I have no idea, I just did a replace all to see how many tests would break. Amazingly none. Will now try and work out what's going on in a bit more detail 😂

Reply:
I guess they haven't actually given us an example of what the new structure will look like in practice. (Unless they've already started sending it and we can look at that.)

Reply:
They have! It's alongside the old one. It looks pretty similar but the names are a bit "cleaner"